server/src/config/__init__.py

```diff
@@ -9,3 +9,5 @@
 # Otherwise, updates would not be reflected if they change the pointer
 def cfg():
     return config_manager.config
+def cfg_last_update():
```

Comment:
I don't think this really warrants a dedicated helper.

The main `cfg()` function is there, because it's needed quite frequently and makes some lines more concise.

Currently there is only 1 location where this new helper would be used, where we can just import `config_manager` and call it directly.